### ISSUE TITLE
Midpoint Review - Design Feedback - Authenticated Experience, My VA, Audit UX Improvements

### ISSUE THREAD

**Issue by allison0034** [department-of-veterans-affairs/va.gov-team#48655]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
-

## Feedback
- Practice areas will document their feedback following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback).

**Must:**
- The green alert and button are reserved for login in only. Change the button to the [primary button](https://design.va.gov/components/button/#primary-button) or if that behaves as navigation, an [action link](https://design.va.gov/components/link/action).
- "What benefits does VA offer?" additional info - it's not a Saved application. Work with IA and find a better home for it on the error state and authenticated state.
- "Education and training" has no "You have no education benefits" state like all of the other sections. It should have that and put the links on the right like the other sections.
- In the error state and in the "You have nothing to show" state the links all slide under the Alerts or under the message. They should stay to the right. No reason to move them. It makes it harder to spot those messages and there is no technical reason that can't be achieved (use the grid correctly).

**Should:**
- Use the new notification style card instead of alert styling for things like "You have 2 unread messages". It is approved and in the backlog to be added to the DS. : )

**Consider:**

## Platform directions
- Remove the governance-team label
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Remove the CC-Request label (only if you created the ticket using the New Issue link from the Collaboration Cycle Reviews board)
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by ajakabcin** [comment]
Hi @shiragoodman - based on the conversation in collab cycle office hours today (11/2) we agreed that the following were no longer considered "must" feedback items and therefore not launch blocking:

> - "What benefits does VA offer?" additional info - it's not a Saved application. Work with IA and find a better home for it on the error state and authenticated state.
> - "Education and training" has no "You have no education benefits" state like all of the other sections. It should have that and put the links on the right like the other sections.
> - In the error state and in the "You have nothing to show" state the links all slide under the Alerts or under the message. They should stay to the right. No reason to move them. It makes it harder to spot those messages and there is no technical reason that can't be achieved (use the grid correctly).

We also received clarification that we are using the green button correctly (or at least as the documentation indicates) and do not need to change this to a blue button.

The team will be prioritizing the build and implementation of the notification component in coming sprints and then will replace the onsite notifications and the inline notification for unread messages in the health care section on My VA with the new component. In both cases, we currently use the "warning alert component" instead.

Thank you!

cc @Samara-Strauss @ericasauv09 @aagosto90

**Comment by humancompanion-usds** [comment]
To clarify, the feedback was to not use a button in that Alert variation but the pushback was that this variation is used across VA.gov in this fashion. We're reviewing that and will, either way, add the variation to the Design System so that the correct usage is documented. We've got [an issue for that in flight](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1298).

> "What benefits does VA offer?" additional info - it's not a Saved application. Work with IA and find a better home for it on the error state and authenticated state.

The resolution on this was that further information would be gathered and the team feels like the link can be removed in future. There was general agreement that it is awkward in this section. 

> "Education and training" has no "You have no education benefits" state like all of the other sections. It should have that and put the links on the right like the other sections.

There was no resolution to this other than it's an accepted state and there is no roadmap for showing education benefits in this section. No changes are forthcoming.

> In the error state and in the "You have nothing to show" state the links all slide under the Alerts or under the message. They should stay to the right. No reason to move them. It makes it harder to spot those messages and there is no technical reason that can't be achieved (use the grid correctly).

This is where we spent the most time understanding the different patterns (they're perhaps different compositions of components, or composed components, but patterns will suffice) at play. I'm documenting them here for my own reference so that I can capture what I hope I now understand. If there is already documentation of these patterns that the team can point me to I would find that helpful for future collaboration.

### A1. Status Card with primary action and Link - Collection
#### Claims and Health care
![Screen Shot 2022-11-03 at 3 22 56 PM](https://user-images.githubusercontent.com/52176351/199815243-a82d2f2d-d57c-419f-896b-fed5c405997f.png)

#### Benefit payments
![Screen Shot 2022-11-03 at 3 27 50 PM](https://user-images.githubusercontent.com/52176351/199816112-c06cfff9-ec12-4758-bfed-3c52ea3d0717.png)

### A2. Status Cards with primary action stacked without Link - Collection
![Screen Shot 2022-11-03 at 3 24 22 PM](https://user-images.githubusercontent.com/52176351/199815484-da8fcd9c-9f99-40c4-a00d-494c37fa5996.png)

This one can be one or more Cards and they stack.

### A3. Status Cards with primary action split without Link - Collection
![Screen Shot 2022-11-03 at 3 25 26 PM](https://user-images.githubusercontent.com/52176351/199815667-1ef85c05-1306-4f42-9c70-24c5b59aa190.png)

This is the same as above and can also have 1 or more cards. But when 2 or more they are split instead of stacked.

### A4. Status Card with secondary links in Additional info
![Screen Shot 2022-11-03 at 3 34 17 PM](https://user-images.githubusercontent.com/52176351/199817325-e374b5e5-2ede-448d-9b2f-00a14a8f494c.png)

This one has the secondary links in Additional info instead of in the Link - Collection.

### B1. Empty status (nothing to show) with Link - Collection

![Screen Shot 2022-11-03 at 3 30 25 PM](https://user-images.githubusercontent.com/52176351/199816606-b1416185-bcce-4cc9-92d8-8ea20635c3a8.png)
![Screen Shot 2022-11-03 at 3 31 34 PM](https://user-images.githubusercontent.com/52176351/199816832-49505993-2428-4251-9d3c-19d718226286.png)
![Screen Shot 2022-11-03 at 3 32 31 PM](https://user-images.githubusercontent.com/52176351/199817030-7f66648f-fb72-44b5-905b-329c72ac536f.png)

### B2. Empty status with secondary links in Additional info
![Screen Shot 2022-11-03 at 4 28 56 PM](https://user-images.githubusercontent.com/52176351/199827498-d4a82531-5a66-4a67-90ba-df9f5dc1bd4c.png)

### C. No status with secondary links
![Screen Shot 2022-11-03 at 3 33 00 PM](https://user-images.githubusercontent.com/52176351/199817111-e3595850-0392-42e5-9f6c-a3d3e7409217.png)

### D1. Error with no secondary links
![Screen Shot 2022-11-03 at 3 47 30 PM](https://user-images.githubusercontent.com/52176351/199819768-3a04cd4f-4b4c-40cd-b347-99770a6acea3.png)

This is because the secondary link would not make sense given the error.

### D2. Error with secondary links
![Screen Shot 2022-11-03 at 4 08 19 PM](https://user-images.githubusercontent.com/52176351/199823751-467eb6aa-0419-4788-8265-4b164578a6fb.png)

Health care, Outstanding debuts, and Benefit payments use this pattern.

### E1. Notification with Status Card with primary action and Link - Collection
![Screen Shot 2022-11-03 at 4 37 07 PM](https://user-images.githubusercontent.com/52176351/199828980-90b2d11f-4aab-4f36-8184-cadfbef96f99.png)

### E2. Notification with empty status with Link - Collection
![Screen Shot 2022-11-03 at 4 40 47 PM](https://user-images.githubusercontent.com/52176351/199829627-532d8963-dc3d-430a-bda9-1fe3a4a016fc.png)

This could be B3 as well. 

## Revised feedback
Now that I've had some time to look at this more carefully and heard from Angela yesterday about the design decisions I would recommend considering the following. You may choose to do with this what you will and at your leisure. I took the time to think it through thus wanted to give you improved feedback.

* Seems like picking between A2 or A3 rather than supporting both might be a place to consolidate.
* In A4 if you do want to keep those links it might make sense to move them below the Cards and use Link - Collection. You could consider splitting the links across the two columns rather than stacking (this is done on Facility pages). "Explore benefits VA has to offer" might warrant a new section at the bottom.
* In B1 and B2, as I mentioned yesterday, the "nothing to show" status message gets a bit lost. Perhaps consider having a Card variation that includes the "nothing to show" text plus a logical primary action instead of the single line of text. As you would in a "no search results" design it would be good to give users a logical next step when there is nothing to show. As it stands the Link - Collection gives a number of potential steps but none is the primary action they might take.

**Comment by aagosto90** [comment]
@humancompanion-usds Thank you for taking the time to look deeper into My VA and send us this feedback. We really appreciate it. We met as a team to discuss and then I played around in Sketch with some options and here's where we landed today:

- We're going to put cards side-by-side if there are multiple within a section, as shown in A3. We already do that with applications and will apply that to the outstanding debt section as well. We looked at both options together and thought side-by-side cards made better use of the vertical space and looked more like a dashboard than if we stacked all cards on the left. We also talked about the fact that it's an edge case that a user would have both copays and debts and/or multiple saved applications. So it's a problem we needed to solve but it's for a small set of users and most people will only see one card per section if any.
- I mocked up a new link collection for all VA benefits at the bottom of My VA under applications and we all agreed it looked busy since it comes right before the gigantic footer that also contains a lot of links. For now we're opting to keep that accordion in the Saved Applications section and we will monitor analytics to see if it declines in popularity. It's our hypothesis that it will with the addition of other calls to action for benefits throughout the page. It's not our intention to leave it there permanently as we agree with you that it doesn't exactly fit with saved applications.
- We agreed as a team that we don't want to put a "nothing to show" message within a card because the cards with dynamic content to show are where we want to draw the user's attention. I personally do kind of want a user to gloss over that "nothing to show" text but want them to notice the secondary link if they're a person who's interested in other VA benefits. We think we'll have some more answers about whether the empty state is a good solution or if we need to revisit a more prominent call to action in those sections coming out of usability testing for these updates so we'll keep an eye out for how veterans react to that.

Let us know if you have any objections to these conclusions. Thank you again!

CC @Samara-Strauss @ajakabcin @ericasauv09
